.travis.yml

```diff
@@ -35,11 +35,9 @@ install:
  - true
 
 script:
-  - ./gradlew build idea -s -PcheckJava6Compatibility
+  - ./gradlew build idea -s -PcheckJava6Compatibility && ./gradlew ciPerformRelease
```

Comment:
Also the other commands like assert and prepare are removed, I guess that is intentional?